app/helpers/mobile_helper.rb

```diff
@@ -9,33 +10,37 @@ def mobile_reshare_icon(post)
       absolute_root = reshare?(post) ? post.absolute_root : post
 
       if absolute_root && absolute_root.author != current_user.person
-        reshare = Reshare.where(:author_id => current_user.person_id,
-                                :root_guid => absolute_root.guid).first
+        reshare = Reshare.where(author_id: current_user.person_id,
+                                root_guid: absolute_root.guid).first
         klass = reshare.present? ? "active" : "inactive"
-        link_to '', reshares_path(:root_guid => absolute_root.guid), :title => t('reshares.reshare.reshare_confirmation', :author => absolute_root.author_name), :class => "image_link reshare_action #{klass}"
+        link_to "", reshares_path(root_guid: absolute_root.guid),
+                title: t("reshares.reshare.reshare_confirmation", author: absolute_root.author_name),
+                class: "entypo-reshare reshare-action #{klass}"
       end
     end
   end
 
   def mobile_like_icon(post)
     if current_user && current_user.liked?(post)
-      link_to '', post_like_path(post.id, current_user.like_for(post).id), :class => "image_link like_action active"
+      link_to "", post_like_path(post.id, current_user.like_for(post).id), class: "entypo-heart like-action active"
     else
-      link_to '', post_likes_path(post.id), :class => "image_link like_action inactive"
+      link_to "", post_likes_path(post.id), class: "entypo-heart like-action inactive"
     end
   end
 
   def mobile_comment_icon(post)
-    link_to '', new_post_comment_path(post), :class => "image_link comment_action inactive"
+    link_to "", new_post_comment_path(post), class: "entypo-comment comment-action inactive"
   end
 
   def reactions_link(post)
     reactions_count = post.comments_count + post.likes_count
     if reactions_count > 0
-      link_to "#{t('reactions', :count => reactions_count)}", post_comments_path(post, :format => "mobile"), :class => 'show_comments'
+      link_to "#{t("reactions", count: reactions_count)}",
+              post_comments_path(post, format: "mobile"),
+              class: "show_comments"
     else
       html = "<span class='show_comments'>"
-      html << "#{t('reactions', :count => reactions_count)}"
+      html << "#{t("reactions", count: reactions_count)}"
```

Comment:
@AugierLe42e This is one of the default rubocop rules. Since the code style comments are only on lines where you only changed the code style, I think the best solution would be to revert those changes.